scripts/post-install.sh

```diff
@@ -8,7 +8,6 @@ LOGROTATE_DIR=/etc/logrotate.d
 if ! id influxdb >/dev/null 2>&1; then
         useradd --system -U -M influxdb -s /bin/false -d $DATA_DIR
 fi
-chown influxdb:influxdb $BIN_DIR/influx*
```

Comment:
Binaries under /usr/bin should be owned by root not an unprivileged user,
that way only root can modify/remove then
On Nov 13, 2015 8:18 PM, "otoolep" <notifications@github.com> wrote:

> In scripts/post-install.sh
> <https://github.com/influxdb/influxdb/pull/4786#discussion_r44854061>:
>
> > @@ -8,7 +8,6 @@ LOGROTATE_DIR=/etc/logrotate.d
> >  if ! id influxdb >/dev/null 2>&1; then
> >          useradd --system -U -M influxdb -s /bin/false -d $DATA_DIR
> >  fi
> > -chown influxdb:influxdb $BIN_DIR/influx*
>
> Can I get some detail on this permission issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/4786/files#r44854061>.
>